packages/components/RadioGroup/src/RadioButton.tsx

```diff
@@ -9,42 +9,52 @@ import { filterViewProps } from '@fluentui-react-native/adapters';
 import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
 import { settings, radioButtonSelectActionLabel } from './RadioButton.settings';
 import { mergeSettings } from '@uifabricshared/foundation-settings';
-import { foregroundColorTokens, textTokens, borderTokens, backgroundColorTokens, getPaletteFromTheme} from '@fluentui-react-native/tokens';
+import { foregroundColorTokens, textTokens, borderTokens, backgroundColorTokens, getPaletteFromTheme } from '@fluentui-react-native/tokens';
 import { useAsPressable, useViewCommandFocus } from '@fluentui-react-native/interactive-hooks';
 import { RadioGroupContext } from './RadioGroup';
 
 export const RadioButton = compose<IRadioButtonType>({
   displayName: radioButtonName,
 
   usePrepareProps: (userProps: IRadioButtonProps, useStyling: IUseComposeStyling<IRadioButtonType>) => {
-    const { content, buttonKey, disabled, ariaLabel, componentRef, ...rest } = userProps;
+    const { content, buttonKey, disabled, ariaLabel, componentRef = React.useRef(null), ...rest } = userProps;
 
     // Grabs the context information from RadioGroup (currently selected button and client's onChange callback)
     const info = React.useContext(RadioGroupContext);
 
+    const buttonRef = useViewCommandFocus(componentRef);
+
     /* We don't want to call the user's onChange multiple times on the same selection. */
     const changeSelection = () => {
-      if(buttonKey != info.selectedKey)
+      if (buttonKey != info.selectedKey) {
         info.onChange && info.onChange(buttonKey);
+        info.updateSelectedButtonRef && componentRef && info.updateSelectedButtonRef(componentRef);
+        componentRef?.current?.focus();
```

Comment:
Possibly? I added this here because radio buttons were not getting keyboard focused after being clicked.  After adding this, keyboard focus would move to the right button after it was clicked (and show the focus border). Was this working before?

Reply:
This was definitely working before, which is weird. If I remember correctly, isn't this happening with other components in our FURN tester? Could this possibly only be an issue with our tester? I just checked and clicking a Checkbox in our tester issues the same keyboard focus error, which definitely didn't happen before.

I'd be curious to see if consumers are running into this issue, or if it's just within our test app.

Reply:
Looks like it's not just our test app but I decided to remove this line anyways since we identified it as a regression across multiple controls. Users can still keyboard to the Radio Group if they can get keyboard focus on some other element and tab to it so this call to focus isn't necessary.